### ISSUE TITLE
Rewrite path of files before backing them up

### ISSUE THREAD

**Issue by pschmitt** [borgbackup/borg#1868]
Hi,

I'm trying to set up backups for my OpenWRT router. Since there is no borg package available, I am doing it through another host by mounting the router via SSHFS. Now the only issue I am facing in this setup is that the files that are backed up have the mountpoint in their path. Is there a way to tell borg to rewrite them before backing them up?

**TLDR**: is there a way to have borg rewrite file paths like `/mnt/remote_host/here/there` to `/here/there`?

**Comment by RonnyPfannschmidt** [comment]
@ThomasWaldmann would it be sensible to have the `--strip-components` idea from  #1650 for all archive creation not just for recreate

**Comment by enkore** [comment]
> TLDR: is there a way to have borg rewrite file paths like /mnt/remote_host/here/there to /here/there?

cd into /mnt/remote_host and do `borg create ... here`?

**Comment by pschmitt** [comment]
@enkore oh. Yeah that actually works. Thanks a lot!

**Comment by ThomasWaldmann** [comment]
add to faq?

**Comment by camlafit** [comment]
Hi

Add this information as sample in http://borgbackup.readthedocs.io/en/stable/usage.html#borg-create looks be great :)
I've search a long time before find this issue and solve same problem.

Thanks a lot

**Comment by enkore** [comment]
#2222
